visualization/dashboard/src/Localization/en.json

```diff
@@ -406,7 +407,9 @@
         "disclaimer": "Disclaimer: These are explanations based on many approximations and are not the \"cause\" of predictions. Without strict mathematical robustness of causal inference, we do not advise users to make real-life decisions based on this tool.",
         "_disclaimer.comment": "the tool should not be liable for any bad predictions",
         "missingParameters": "This tab requires an evaluation dataset be supplied.",
-        "_missingPArameters.comment": "Show a message if the required dataset parameter is not provided"
+        "_missingParameters.comment": "Show a message if the required dataset parameter is not provided",
+        "selectionLimit": "Maximum of 3 selected points",
+        "_selectionLimit.comment": "A user can only select 3 points from a chart at a time, this message is displayed if they click a 4th"
```

Comment:
Will this message be shown directly to users? If so, might be best to direct it at them

Reply:
ok now I'm seeing the pattern in this file with the underscore and .comment. Now I'm just curious - how does this show up for developers?

Reply:
its for translators, they see it as a comment in the file they are translating next to the string to be translated